packages/lodestar-validator/src/types.ts

```diff
@@ -1,14 +1,18 @@
 /**
  * @module validator
  */
-import {AttesterDuty} from "@chainsafe/lodestar-types";
+import {SecretKey} from "@chainsafe/bls";
+import {AttesterDuty, ValidatorResponse} from "@chainsafe/lodestar-types";
 
 export type GenesisInfo = {
   startTime: number;
 };
 
 export interface IAttesterDuty extends AttesterDuty {
-  // internal index
-  attesterIndex: number;
   isAggregator: boolean;
 }
+
+export type ValidatorAndSecret = {
+  validator: ValidatorResponse | null;
```

Comment:
why is this `| null`?

Reply:
no good reason.  i think this is a vestige of a previous step of this PR.  will remove the `| null`

Reply:
oh wait, actually, it's because this is what was there before.  might actually need to put this back as the tests are failing after i changed it:
https://github.com/ChainSafe/lodestar/pull/1890/files#diff-ff30b1253b91df678666c2f076e07037711e7bac916b98fc0257c4ee7046339eL51